src/components/table/Table.stories.tsx

```diff
@@ -0,0 +1,433 @@
+                         <td>Sometimes</td>
+                         <td className="left">Not really</td>
+                     </tr>
+                     <tr>
+                         <td className="left">Mothra</td>
+                         <td>Tropical island</td>
+                         <td>No</td>
+                         <td className="left">Only if you have a flashlight</td>
+                     </tr>
+                     <tr>
+                         <td className="left">Oscar the Grouch</td>
+                         <td>Sesame Street</td>
+                         <td>No</td>
+                         <td className="left">No</td>
+                     </tr>
+                     <tr>
+                         <td className="left">Cookie Monster</td>
+                         <td>Sesame Street</td>
+                         <td>Definitely yes</td>
+                         <td className="left">Only if you have no cookies</td>
+                     </tr>
+                 </tbody>
+             </Table>
+         </div>
+     ))
+     .add("Compact, Non-Bordered Tables", () => (
+         <div style={{width: "50%", paddingLeft: "20px"}}>
+             <Table isCompact={true} isNonBordered={true}>
+                 <thead>
+                     <tr>
+                         <th className="left">Monster</th>
+                         <th>Home</th>
+                         <th>Likes Cookies</th>
+                         <th className="left">Fun to Play With</th>
+                     </tr>
+                 </thead>
+                 <tbody>
+                     <tr>
+                         <td className="left">Wolfman</td>
+                         <td>Nondisclosed countryside</td>
+                         <td>Sometimes</td>
+                         <td className="left">Not really</td>
+                     </tr>
+                     <tr>
+                         <td className="left">Mothra</td>
+                         <td>Tropical island</td>
+                         <td>No</td>
+                         <td className="left">Only if you have a flashlight</td>
+                     </tr>
+                     <tr>
+                         <td className="left">Oscar the Grouch</td>
+                         <td>Sesame Street</td>
+                         <td>No</td>
+                         <td className="left">No</td>
+                     </tr>
+                     <tr>
+                         <td className="left">Cookie Monster</td>
+                         <td>Sesame Street</td>
+                         <td>Definitely yes</td>
+                         <td className="left">Only if you have no cookies</td>
+                     </tr>
+                 </tbody>
+             </Table>
+         </div>
+     ))
+     .add("Vertical Tables", () => (
+         <div style={{width: "50%", paddingLeft: "20px"}}>
+             <Table isVertical={true}>
+                 <tbody>
```

Comment:
some stories are suing  `<thead> `and some are not. any reason ? what is recommended/standard way to render table ?

Reply:
<thead> is not there for vertical tables only. So for the vertical table our structure is like <tr><th /><td /></tr>. We are creating a row in which first column is the header and rest columns contains data.